sentinel-core/src/main/java/com/alibaba/csp/sentinel/fallback/DefaultGlobalFallback.java

```diff
@@ -0,0 +1,16 @@
+package com.alibaba.csp.sentinel.fallback;
```

Comment:
Could you please prepend the license header in the new files?

Reply:
> Could you please prepend the license header in the new files?

Completed in the last commit